app/policies/article_policy.rb

```diff
@@ -200,11 +200,10 @@ def permitted_attributes
   private
 
   def user_author?
-    if record.instance_of?(Article)
-      record.user_id == user.id
-    else
-      record.pluck(:user_id).uniq == [user.id]
-    end
+    # We might have the Article class (instead of the Article instance), so let's short circuit
```

Comment:
Note in a previous implementation (introduced in https://github.com/forem/forem/pull/630/files), we were querying a subset of Article.  That implementation (e.g. `ArticlePolicy#analytics_index?`) is gone.